forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/FormBPMDataPipelineListener.java

```diff
@@ -20,6 +22,8 @@
 @Named("FormBPMDataPipelineListener")
```

Comment:
Moving out from Java -DI to Spring DI can be considered.

Reply:
Yes, Of course. `@Named` support has been added in Spring `3.0,` we are now using `2.7.12` version.